chisel/src/cmd_run.rs

```diff
@@ -110,7 +110,7 @@ pub fn chisel_run(flags: ChiselFlags) -> i32 {
     eprintln!("{}", &results);
     results
         .rulesets_mut()
-        .into_iter()
+        .iter_mut()
```

Comment:
I think I decided to explicitly consume by value here but it is a matter of personal preference.